common/model-views.h

```diff
@@ -521,16 +526,24 @@ namespace rs2
             rs2::frame f{};
             model = f;
             while (resulting_3d_models.poll_for_frame(&f));
-            while (depth_frames_to_render.poll_for_frame(&f));
+        }
+        void start()
```

Comment:
It lacks stop() functionality, as in case the streaming is stopped the PC oricessing shall be stopped